### PR TITLE
Add BTC deposit test for more coverage 

### PR DESCRIPTION
**Pull request by jo-tm**
test-minimum-burnchain-confirmations-not-reached

### CODE REVIEW

File: sbtc-mini/tests/sbtc-peg-in-processor_test.clar [hunk 1]
```diff
@@ -230,3 +232,27 @@
...
+                     mock-burnchain-height
+                     mock-tx-1
+                     mock-block-header-1
+                     u1
+                     u1
+                     (list mock-coinbase-wtxid-1)
+                     mock-witness-root-hash-1-le
+                     mock-coinbase-witness-reserved-data
+                     mock-witness-index-1
+                     (concat mock-coinbase-tx-1 0x) ;; Appending a zero byte to make the script invalid
```
Comment (friedger) [inline sbtc-mini/tests/sbtc-peg-in-processor_test.clar:248]: The comment is not related to the test, is it?

**Review by friedger — COMMENTED** [review]
Why is there only 1 check in this PR?

Otherwise, looks good, just a small nit

File: sbtc-mini/tests/sbtc-peg-in-processor_test.clar (hunk 1 above)
Comment (jo-tm) [reply sbtc-mini/tests/sbtc-peg-in-processor_test.clar]: removed the old comment.

**Comment by jo-tm** [comment]
@friedger do you mean you want more test cases for the same error?

**Comment by friedger** [comment]
@jo-tm there is something wrong with the CI? 

Screenshot_20230815-175055

**Comment by jo-tm** [comment]
@friedger I think we just changed the name of the files: `peg-in` -> `deposit`

**Comment by jo-tm** [comment]
@friedger this is Green now.

File: sbtc-mini/tests/sbtc-deposit-processor_test.clar [hunk 2]
```diff
@@ -230,3 +232,23 @@
 	)
 )
 
+;; Test case for minimum burnchain confirmations not reached
```
Comment (MarvinJanssen) [inline sbtc-mini/tests/sbtc-deposit-processor_test.clar:235]: Can you use the `@name` annotation?

I also just realised annotations are missing for most tests. I will update those.

**Review by MarvinJanssen — APPROVED** [review]
Looks good!

**Review by friedger — APPROVED** [review]
Looks good.
LGTM

**Pull request merged**